examples/simple-platform/platform-structure.md

```diff
@@ -0,0 +1,45 @@
+# Platform structure
+
+Our platform has two server nodes. One runs the `platform-deploy` hypervisor 
+and the other runs the `web-auth-endpoint` hypervisor. We also have two 
+browser nodes - one for the accounts app and another for the template app. 
+These nodes run the React apps in the browser.
+
+The `platform-deploy` node hosts the accounts and template chains, acting as 
+the server for the two apps. It is the blockmaster and thus is responsible for 
+adding new blocks to the application chains. Note that there is no requirement 
+for the accounts and template chains to be hosted on the same node. They could 
+be hosted on separate nodes and moreover each app's chains could be hosted on 
+multiple nodes for redundancy. (FIXME: Should we mention that this has not 
+been implemented yet?) 
```

Comment:
I think it's fine not to mention it. We're just stating that it's possible, not that we're doing it in practice.